hack/inventory/test-sample/rook/cluster.yaml

```diff
@@ -8,55 +8,45 @@
 #   kubectl create -f operator.yaml
 #   kubectl create -f cluster.yaml
 #################################################################################################################
-kind: ConfigMap
```

Comment:
As far as I know, for the test environment (k8s cluster installed with less than 3 nodes), this configmap is necessary. And it seems like `mon_warn_on_pool_no_redundancy = false` is added since rook v1.4.2
https://github.com/rook/rook/blob/v1.4.2/cluster/examples/kubernetes/ceph/cluster-test.yaml